frontend/src/components/about/Contact.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <about-layout title="License">
+  <about-layout title="Contacts">
```

Comment:
I think I prefer one of these alternatives better:

```suggestion
  <about-layout title="Contact">
```

or 

```suggestion
  <about-layout title="Contact us">
```

Reply:
Changed in the commit f1c3273d

Besides, I've also added the description for how to report bugs and vulnerabilities as well as how to use management@metabolicatlas.org